crossplane.yaml

```diff
@@ -36,15 +36,17 @@ spec:
   crossplane:
     version: ">=v1.12.1-0"
   dependsOn:
-    - provider: xpkg.upbound.io/upbound/provider-family-azure
-      version: ">=v0.33.0"
-    - provider: xpkg.upbound.io/upbound/provider-azure-containerservice
-      version: ">=v0.33.0"
-    - provider: xpkg.upbound.io/upbound/provider-azure-dbformariadb
-      version: ">=v0.33.0"
-    - provider: xpkg.upbound.io/upbound/provider-azure-dbforpostgresql
-      version: ">=v0.33.0"
-    - provider: xpkg.upbound.io/upbound/provider-azure-network
-      version: ">=v0.33.0"
-    - provider: xpkg.upbound.io/crossplane-contrib/provider-helm
-      version: ">=v0.15.0"
```

Comment:
It looks like we are changing the indentation style for arrays in this whole configuration. it does not look consistent with https://github.com/upbound/platform-ref-aws/blob/main/crossplane.yaml#L32
Can we avoid reindentation where it is not needed?